ext/pycall/pycall.c

```diff
@@ -1165,6 +1172,7 @@ pycall_libpython_helpers_m_getitem(VALUE mod, VALUE pyptr, VALUE key)
 
   obj = pycall_pyobject_to_ruby(pyobj_v);
   pycall_Py_DecRef(pyobj_v);
+  pycall_Py_DecRef(pyobj_key);
```

Comment:
This `DecRef` of `pyobj_key` should be called just after calling `PyObject_GetItem`.